doc/api/child_process.markdown

```diff
@@ -388,29 +97,34 @@ Runs a command in a shell and buffers the output.
         }
     });
 
-The callback gets the arguments `(error, stdout, stderr)`. On success, `error`
-will be `null`.  On error, `error` will be an instance of [`Error`][] and `error.code`
-will be the exit code of the child process, and `error.signal` will be set to the
+If a `callback` function is provided, it is called with the arguments
+`(error, stdout, stderr)`. On success, `error` will be `null`.  On error,
+`error` will be an instance of [`Error`][]. The `error.code` property will be
+the exit code of the child process while `error.signal` will be set to the
 signal that terminated the process.
 
-There is a second optional argument to specify several options. The
-default options are
+The `options` argument may be passed as the second argument to customize how
+the process is spawned. The default options are:
 
-    { encoding: 'utf8',
+    {
+      encoding: 'utf8',
       timeout: 0,
       maxBuffer: 200*1024,
       killSignal: 'SIGTERM',
       cwd: null,
-      env: null }
+      env: null
+    }
 
-If `timeout` is greater than 0, then it will kill the child process
-if it runs longer than `timeout` milliseconds. The child process is killed with
-`killSignal` (default: `'SIGTERM'`). `maxBuffer` specifies the largest
-amount of data (in bytes) allowed on stdout or stderr - if this value is
-exceeded then the child process is killed.
+* If `timeout` is greater than `0`, then the child process will be terminated
```

Comment:
It might be better to keep the original `kill` terminology so it's clearer how the process is terminated.